backend/app/services/adapters/executor_kinds.py

```
@@ -408,6 +408,28 @@ def _format_subtasks_response(
                 )
 
+                # If user shell not found, try public shells
+                shell_base_image = None
+                if not shell:
+                    from app.models.public_shell import PublicShell
+
+                    public_shell = (
+                        db.query(PublicShell)
+                        .filter(
+                            PublicShell.name == bot_crd.spec.shellRef.name,
+                            PublicShell.is_active == True,
```

Ruff (0.14.6)

Avoid equality comparisons to `True`; use `PublicShell.is_active:` for truth checks

Replace with `PublicShell.is_active`

(E712)